docs/integrations/conda.md

```diff
@@ -1,3 +1,8 @@
 # Anaconda Integration
 
 Page under Construction
+
+```eval_rst
+.. meta::
+   :description: Jovian.ml integration with anaconda.org
```

Comment:
Jovian integration with Anaconda. 

Let's use "Jovian" (instead of Jovian.ml) and punctuation at the end for consistency.